### PR TITLE
Bugfix && improvements

### PR DESCRIPTION
**Pull request by falkolab**
- findScrollableView argument (true by default)
- bug fixes
- improvements

### CODE REVIEW

**Comment by falkolab** [comment]
In my project i have separate ScrollableView and controller adds Views programmatically.

File: de.manumaticx.pagingcontrol/controllers/widget.js [hunk 1]
```diff
@@ -188,6 +194,9 @@ function adjustePositions() {
  * @param {Ti.UI.Scrollableview} scrollable view
  */
 exports.setScrollableView = function(_sv){
+    if($.scrollableView) {
+        throw "Already initialized";
+    }
```
Comment (manumaticx) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: the user may want to re-initialize the widget with another instance. why do we need this?

File: de.manumaticx.pagingcontrol/controllers/widget.js [hunk 2]
```diff
@@ -7,7 +7,13 @@ _.defaults(args, {
...
     scrollOffset: 40,
     height: args.tabs ? 48 : 5,
-    width: Ti.UI.FILL
+    width: Ti.UI.FILL,
+    findScrollableView: true
+});
+
+// xml boolean args is string ("false" == true) 
+_.each(['tabs', 'findScrollableView'], function(key){
+    args[key] = JSON.parse(args[key]);
```
Comment (manumaticx) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: JSON.parse is throwing an exeption. Can you change this to `!!args[key]`?

Comment (falkolab) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: it should not cause exception
```
var args={
  param: 'false'
};

!!args.param // true
```

**Comment by manumaticx** [comment]
Thanks for your contribution! I've commented on the commit. 
Can you please change these parts or add a more specific explanation why the setter needs to be changed and the `findScrollableView` is used for? Thanks!

**Comment by falkolab** [comment]
Setter:
Subsequent call setter after auto set scrollableView cause unpredictable result if I remember correctly.

findScrollableView:
Used for disable auto set scrollableView if my separate scrollableView initially empty and will be filled later with widget setter.

File: de.manumaticx.pagingcontrol/controllers/widget.js (hunk 2 above)
Comment (manumaticx) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: ok, can we surround it with try-catch then, and only apply if it's a valid boolean (or boolean string)?

**Comment by falkolab** [comment]
setter cause multiply calls to init function. Memory leaks with event handlers on old tabs?

File: de.manumaticx.pagingcontrol/controllers/widget.js (hunk 1 above)
Comment (falkolab) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: setter cause multiply calls to init function. Memory leaks with event handlers on old tabs?

**Comment by manumaticx** [comment]
Ok, can we replace the `throw` with `Ti.API.error` and `return;`?

**Comment by falkolab** [comment]
ok

File: de.manumaticx.pagingcontrol/controllers/widget.js (hunk 2 above)
Comment (falkolab) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: Fixed

Comment (falkolab) [inline de.manumaticx.pagingcontrol/controllers/widget.js]: Oh one moment please.

**Comment by manumaticx** [comment]
Thank you :)

**Pull request merged**